pnp/PnpHelper.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+
+namespace PnpHelpers
+{
+    public static class PnpHelper
```

Comment:
Does this helper suppose to have PnP_Sample_Helper_Parse or did I miss something?

Reply:
I don't know what you mean by that, can you elaborate on it? What is `PnP_Sample_Helper_Parse`?

Reply:
It is in the spec if you see.

Reply:
Ah, you mean, this from the specs doc:
```C
ampleTwinCb(JSON update) { 
PnP_Sample_Helper_Parse(update, ...); 
//"Component:Foo", "Pro:bar", "val" 
 
char* jsonToSend = PnP_Helper_CreateReported("foo", "bar", 200, value); 
moduleClient.SendProprety(jsonToSend); 
}
```

This is just a sample code on how the helper is expected to work. We don't need to follow it word-to-word. 
What this does mean is, we need a helper that can create a pnp-compatible reported property patch; which is something we will have under `PnpHelper`.

Also, I would like to re-iterate, this is work-in-progress, so not all helpers are implemented yet. That being said, feel free to leave your comments on the ones that have been implemented, all feedback is much-appreciated! :)